### PR TITLE
Remove hardcoded path for cnvkit filtering.

### PR DESCRIPTION
**Pull request by Smeds**

### CODE REVIEW

**Comment by Smeds** [comment]
* Will move helper functions to lib and add some test. 
* Also want to test run and make sure that the plots are created.

**Comment by Smeds** [comment]
I will need some help with this one next week @jonca79 . I don't manage to get region-plots to work and we also need to make sure that I have the correct cnv kit NoP file.

File: src/Snakemake/rules/CNV/cnvkit.smk [hunk 1]
```diff
@@ -78,3 +78,146 @@ rule Filter_cnv:
...
+rule create_gene_plots:
+    input:
+        cns="CNV/cnvkit_calls/{sample}-ready.cns",
+        cnr="CNV/cnvkit_calls/{sample}-ready.cnr",
+        relevant_genes=config["cnvkit"]["relevant_genes"],
+        cnv_kit_bed="CNV/bed/cnvkit_manifest.target.bed",
+    output:
+        png="Results/DNA/CNV/{sample}_{gene}_{chrom}:{gene_region1}.png",
+    params:
+        extra="-y-min -2 --y-max 2",
```
Comment (jonca79) [inline src/Snakemake/rules/CNV/cnvkit.smk:109]: --y-min -2 --y-max 2

File: src/Snakemake/rules/CNV/cnvkit.smk [hunk 2]
```diff
@@ -78,3 +78,146 @@ rule Filter_cnv:
...
+
+
+rule create_cnv_kit_plots:
+    input:
+        cns="CNV/cnvkit_calls/{sample}-ready.cns",
+        cnr="CNV/cnvkit_calls/{sample}-ready.cnr",
+    output:
+        png="Results/DNA/CNV/{sample}.png",
+    params:
+        extra="-y-min -2 --y-max 2",
```
Comment (jonca79) [inline src/Snakemake/rules/CNV/cnvkit.smk:90]: --y-min -2 --y-max 2

File: src/Snakemake/rules/CNV/cnvkit.smk (hunk 1 above)
Comment (jonca79) [inline src/Snakemake/rules/CNV/cnvkit.smk:109]: Remove

File: src/Snakemake/rules/CNV/cnvkit.smk [hunk 3]
```diff
@@ -78,3 +78,146 @@ rule Filter_cnv:
...
+rule create_gene_region_plots:
+    input:
+        cns="CNV/cnvkit_calls/{sample}-ready.cns",
+        cnr="CNV/cnvkit_calls/{sample}-ready.cnr",
+        relevant_genes=config["cnvkit"]["relevant_genes"],
+        cnv_kit_bed="CNV/bed/cnvkit_manifest.target.bed",
+    output:
+        png="Results/DNA/CNV/{sample}_{gene}_{chrom}:{gene_region1}.png",
+    params:
+        extra="-y-min -2 --y-max 2",
```
Comment (jonca79) [inline src/Snakemake/rules/CNV/cnvkit.smk:133]: Remove

**Pull request merged**